File: baekjoon/2775/2775.py
'''
부녀회장이 될테야 python 풀이
https://doctcoder.tistory.com/entry/%EB%B6%80%EB%85%80%ED%9A%8C%EC%9E%A5%EC%9D%B4-%EB%90%A0%ED%85%8C%EC%95%BC-%ED%92%80%EC%9D%B4
'''
apt=[[0 for col in range(15)] for row in range(15)]
apt[0]=[x+1 for x in range(15)]
for i in range(1,15):
	for j in range(15):
		for k in range(j+1):
			apt[i][j]+=apt[i-1][k]
t=int(input())
for i in range(t):
	k=int(input())
	n=int(input())
	print(apt[k][n-1])

File: baekjoon/2775/2775.cpp
/*
부녀회장이 될테야 C++ 풀이
https://doctcoder.tistory.com/entry/%EB%B6%80%EB%85%80%ED%9A%8C%EC%9E%A5%EC%9D%B4-%EB%90%A0%ED%85%8C%EC%95%BC-%ED%92%80%EC%9D%B4
*/
#include <iostream>

using namespace std;
int aprt[15][15]={0}; 
int main()
{
  for(int j=0;j<15;j++) // 0층에 있는 사람 구함
  {
    aprt[0][j]=j+1;
  }
  for(int a=1;a<15;a++) // 1층부터 14층까지
  {
    for(int b=0;b<15;b++) // 0호부터 14호까지
    {
      for(int c=0;c<=b;c++) // 어느 호에 몇 명이 사는지 계산
      {
        aprt[a][b]+=aprt[a-1][c];
      }
    }
  }
  int t;
  cin>>t;
  for(int i=0;i<t;i++)
  {
    int k,n;
    cin >> k;
    cin >> n;  
    cout << aprt[k][n-1]<<"\n"; // 출력
  }
}
